WordPress/Sniffs/VIP/PostsPerPageSniff.php

```diff
@@ -20,10 +20,21 @@
  *
  * @since   0.3.0
  * @since   0.13.0 Class name changed: this class is now namespaced.
+ * @since   0.14.0 Added the posts_per_page property.
  */
 class PostsPerPageSniff extends AbstractArrayAssignmentRestrictionsSniff {
 
 	/**
+	 * Posts per page property
+	 *
+	 * Posts per page limit to check against.
+	 *
+	 * @var int
+	 * @since 0.14.0
```

Comment:
Minor nitpick: the `@var` tag should be the last in the doc block and should have one blank line above it, so:
```php
 * @since 0.14.0
 * 
 * @var int
 */
```

Reply:
No problem will fix it now :)